docs/src/references/agent/software-management.md

```diff
@@ -4,29 +4,142 @@ tags: [Reference, Agent, Software Management]
 sidebar_position: 3
 ---
 
-Thin-edge software management is implemented by the `software-update` operation,
-which gives the ability to:
-- manage software packages of different types on the same device 
-- fetch a pertinent subset of the software packages installed on a device
-- install, update and remove software packages on a device
+Thin-edge software management is implemented by two operations
+which give the ability to manage software packages of different types on the same device.
 
-## MQTT API
+- `software_list` is used to fetch a pertinent subset of the software packages installed on a device.
+- `software_update` let the user install, update and remove software packages on a device.
 
-The `software-update` operation API follows the [generic thin-edge rules for operations](device-management-api.md):
-- The `te/<device-topic-id>/cmd/software-update` topic is used to publish the type of software packages
+## `software_list` MQTT API
```

Comment:
chore: code block in header
https://thin-edge.github.io/thin-edge.io/contribute/writing-documentation/markdown-guidelines/#avoid-in-line-code-blocks-in-headers